downstream/modules/platform/proc-migrate-playbooks-roles.adoc

```diff
@@ -0,0 +1,34 @@
+
+[id="proc-migrate-playbooks-roles_{context}"]
+
+
+= Migrating your Ansible playbooks and roles to Core 2.12
+
+[role="_abstract"]
+
+When you are migrating from non collection-based content to collection-based content, you should use Fully Qualified Collection Names (FQCN) when using Ansible content collections in playbooks and roles. If the collection name is not defined, you may encounter behavioral changes within your environment.
```

Comment:
```suggestion
When you are migrating from non collection-based content to collection-based content, you should use the Fully Qualified Collection Names (FQCN) in playbooks and roles to avoid unexpected behavior.
```